app/src/main/java/com/example/nvdovin/weatherapp/presentation/details/adapter/NextDaysRecyclerAdapter.java

```diff
@@ -0,0 +1,103 @@
+import com.example.nvdovin.weatherapp.domain.model.DailyForecast;
+import com.example.nvdovin.weatherapp.domain.utils.date.DateConvertor;
+import com.example.nvdovin.weatherapp.domain.utils.sharedpreferences.SharedPrefs;
+import com.example.nvdovin.weatherapp.domain.utils.temperature.TemperatureConvertor;
+import com.example.nvdovin.weatherapp.domain.utils.time.TimeUtils;
+
+import java.util.Date;
+import java.util.List;
+
+import butterknife.BindView;
+import butterknife.ButterKnife;
+
+class NextDaysRecyclerAdapter extends RecyclerView.Adapter<NextDaysRecyclerAdapter.NextDaysViewHolder> {
+
+    private static final int NUMBER_OF_DAYS_TO_FORECAST = 4;
+    private static final String DATE_FORMAT = "EEEE";
+    private static final int TOMMOROW_ADD_DAY = 1;
+    private static final int YESTERDAY_SUBSTRACT_DAY = -1;
+    private List<DailyForecast> dailyForecastList;
+    private DateConvertor dateConvertor;
+    private SharedPrefs sharedPrefs;
+    private Context context;
+
+
+    NextDaysRecyclerAdapter(List<DailyForecast> dailyForecastList, Context context) {
+        this.dailyForecastList = dailyForecastList;
+        dateConvertor = new DateConvertor();
+        sharedPrefs = new SharedPrefs(context);
+        this.context = context;
+    }
+
+    @Override
+    public NextDaysViewHolder onCreateViewHolder(ViewGroup parent, int viewType) {
+        View view = LayoutInflater.from(parent.getContext()).inflate(R.layout.detail_five_days_recycler_row, parent, false);
+        return new NextDaysViewHolder(view);
+    }
+
+    @Override
+    public void onBindViewHolder(NextDaysViewHolder holder, int position) {
+        DailyForecast dailyForecast = dailyForecastList.get(position);
+        holder.name.setText(getDayString(dailyForecast.getDate()));
+        holder.temperature.setText(TextUtils.concat(
+                TemperatureConvertor
+                        .fromId(sharedPrefs.getTempScaleFromPrefs())
+                        .convertToTemperature(dailyForecast.getDayTempMin()), " ~ ",
+
+                        TemperatureConvertor
+                                .fromId(sharedPrefs.getTempScaleFromPrefs())
+                                .convertToTemperature(dailyForecast.getDayTempMax()))
+        );
+    }
+
+    @Override
+    public int getItemCount() {
+        return NUMBER_OF_DAYS_TO_FORECAST;
+    }
+
+    class NextDaysViewHolder extends RecyclerView.ViewHolder {
+        @BindView(R.id.detail_five_days_day_name)
+        TextView name;
+        @BindView(R.id.detail_five_days_temperatures)
+        TextView temperature;
+
+        NextDaysViewHolder(View itemView) {
+            super(itemView);
+            ButterKnife.bind(this, itemView);
+        }
+    }
+
+    public String getDayString(Date dateToCheck){
+        String dayString = "";
+        Date todayDate = TimeUtils.getTodayDate();
+
+        if (TimeUtils.isSameDate(dateToCheck, TimeUtils.addDaysToDate(todayDate, YESTERDAY_SUBSTRACT_DAY)))
+            {
```

Comment:
please change code style for if/else to have the brakets in the same line
if(){
}else{
}